src/forgot-password/messaging/forgot-password.ts

```diff
@@ -0,0 +1,20 @@
+import buildMessageTemplate from "../../../lib/emails/buildMessageTemplate";
```

Comment:
Please move this file to lib/forgot-password/messaging/forgot-password.ts - I personally also prefer a src folder but honestly the refactor is not worth it.